source/transfers/contracts/TransferHandlerRegistry.sol

```diff
@@ -0,0 +1,41 @@
+pragma solidity 0.5.12;
+
+import "./interfaces/ITransferHandler.sol";
+
+/**
+  * @title TransferHandlerRegistry: holds registry of contract to
+  * facilitate token transfers
+  */
+contract TransferHandlerRegistry {
+
+  event AddTransferHandler(
+    bytes4 kind,
+    address contractAddress
+  );
+
+  // Mapping of bytes4 to contract interface type
+  mapping (bytes4 => ITransferHandler) private _transferHandlerMapping;
```

Comment:
I find it weird to have the word `mapping` int he name of our mapping, we don't normally do that. Maybe I'm being picky though.

Reply:
We normally name it the plural form of what it is. i.e. `indexes`, `rules` etc

Reply:
ok can change